src/PDFA-3-0/Cert/XIS-Server/medmij-pdfa-xis-2-5-serve-1-pdfa.xml

```diff
@@ -7,6 +7,10 @@
     <name value="PDFA - XIS Server - Scenario 2.5 - Serve one PDFA document"/>
     <description value="Scenario 2.5 - Serve one PDFA document of XXX-Schulte via a HTTP read. If PDFA documents are served through Binary resources, scenario 1.4 should be executed instead."/>
     <nts:patientTokenFixture href="pdfa-nl-core-patient-XXX-Schulte-token.xml"/>
+    
+    <profile id="OperationOutcome-profile">
```

Comment:
@yvettevdhaas , zou je hier nog een nts:profile van kunnen maken ivm. de onderhoudbaarheid van onze materialen?
https://github.com/Nictiz/Nictiz-tooling-testscripts/tree/main/generate#defining-fixtures-profiles-and-rules

Reply:
Vergelijkbare constructies zitten in scenario 1.3, 1.4 en 1.5, dus die kunnen dan ook meteen meegenomen worden.